<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forecast_Tab5</class>
 <widget class="QWidget" name="Forecast_Tab5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="Forecast_Tab5_CurrentConfigurationLBL">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Current configuration:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="Forecast_Tab5_CurrentConfigurationTE"/>
   </item>
   <item>
    <widget class="QLabel" name="Forecast_Tab5_ProgressBarLBL">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Progress information:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="Forecast_Tab5_ProgressBarPBR">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_53">
     <item>
      <widget class="QPushButton" name="Forecast_Tab5_PrevPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Go to previous tab page</string>
       </property>
       <property name="statusTip">
        <string>Go to previous tab page</string>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_45">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Forecast_Tab5_RunPB">
       <property name="toolTip">
        <string>Run the current Forecast configuration</string>
       </property>
       <property name="statusTip">
        <string>Run the current Forecast configuration</string>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_42">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>layoutWidget_2</zorder>
  <zorder>Forecast_Tab5_CurrentConfigurationLBL</zorder>
  <zorder>Forecast_Tab5_ProgressBarPBR</zorder>
  <zorder>Forecast_Tab5_CurrentConfigurationTE</zorder>
  <zorder>Forecast_Tab5_ProgressBarLBL</zorder>
  <zorder>Forecast_Tab5_ProgressBarLBL</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
